Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2047,9 +2047,15 @@ - (void)setMinimumViewportInset:(CocoaEdgeInsets)minimumViewportInset maximumVie
     _maximumViewportInset = maximumViewportInset;
 }
 
-#if USE(APPLE_INTERNAL_SDK)
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/GameControllerSoftLinkAdditions.h>)
```

Comment:
The `__has_include` doesn't seem quite right here, given this is a pre-existing import.

Reply:
It is not a preexisting import (I can point you to the additions patch as well if you'd like)

Reply:
The import already exists on `main` though: https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm#L2051

Reply:
Apologies, I misunderstood - Without the context of the literal next line showing, I thought you were saying "<WebKitAdditions/GameControllerSoftLinkAdditions.h> is a preexisting import"

Yes, this was wrong. New patch coming.